src/include/sof/audio/smart_amp/smart_amp.h

```diff
@@ -73,6 +94,24 @@ struct smart_amp_buf_struct_t {
 	struct smart_amp_fb_buf_struct_t fb;
 };
 
+struct param_buf_struct_t {
+	int id;
+	int value;
+};
+
+struct smart_amp_caldata {
+	uint32_t data_size;
+	void *data;
+	uint32_t data_pos;
+};
+
+struct smart_amp_param_struct_t {
+	struct param_buf_struct_t param;
+	struct smart_amp_caldata caldata;
+	int pos;
+	int max_param;
+};
+
```

Comment:
Can we have some comments for these structures.

Reply:
Shall add comment for each member variable. Thanks.